convert-to-lowercase/convertToLowercase.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+find . -type f -name '*' -print0 | while IFS= read -r -d '' file; do
+#Check to see if the filename contains any uppercase characters
```

Comment:
I would typically align these comments with the code around them rather than having no indent, and put a space after the # for increased readability.